### PR TITLE
add css to the body

### PR DESCRIPTION
**Pull request by guynuriel**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
great

File: dist/style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body{
+    background-color: aqua
```
Comment (morbargig) [inline dist/style.css:2]: good job

**Review by morbargig — CHANGES_REQUESTED** [review]
i want you to go to the beach

File: dist/style.css (hunk 1 above)
Comment (OmerBarashh) [inline dist/style.css:2]: good shit

**Review by OmerBarashh — APPROVED** [review]

File: dist/style.css (hunk 1 above)
Comment (guynuriel) [reply dist/style.css:2]: yes
 im good one

**Review by guynuriel — COMMENTED** [review]
gbkb

File: dist/style.css (hunk 1 above)
Comment (ghost) [reply dist/style.css:2]: great

**Review by ghost — COMMENTED** [review]
great jpbbb

**Pull request merged**